main/EDInitMod.F90

```diff
@@ -201,6 +202,7 @@ subroutine zero_site( site_in )
 
     ! fusoin-induced growth flux of individuals
     site_in%growthflux_fusion(:,:) = 0._r8
+!    site_in%ageflux_fusion(:,:) = 0._r8
```

Comment:
delete?

Reply:
Oh, I forgot about that. I'll delete all that ageflux code since it is not meaningful.